content/doc/marketplace/_index.md

```diff
@@ -148,38 +148,6 @@ Response Status: 200
 
 * `addon_id` - This is the same as the `id` field set in the response to the provisioning call.
 
```

Comment:
There are also mentions of plan change in other parts of the page (Line 65 for example)

Reply:
Right, it is done. All references to "plan" do not relate to plan change except for the removed one from line 65.